sap/rfc/user.py

```diff
@@ -296,6 +375,28 @@ def _call_bapi_method(self, connection, method_name, kwargs):
         BAPIError.raise_for_error(resp['RETURN'], resp)
         return resp
 
+    def _call_user_change_password(self, connection, params):
+        return self._call_bapi_method(connection, 'SUSR_USER_CHANGE_PASSWORD_RFC', params)
+
+    def _fetch_user_type(self, connection, username: UserId) -> str:
+        bapi_return = self.fetch_user_details(connection, username)
+        return bapi_return.get('LOGONDATA', {}).get('USTYP', '')
```

Comment:
What about adding logs about missing LOGONDATA and USTYP? I think that would simplify solving problems.

Reply:
Logging was added.